### PR TITLE
[FEATURE] Restructure Main Menu

### PR DESCRIPTION
**Pull request by susannemoog**
image

### CODE REVIEW

**Comment by linawolf** [comment]
I think the two Points Developing & Using Extensions and then Developing with TYPO3 are kind of confusing. How can you Develop with TYPO3 without Extensions? Or is the second one about Core Development?

**Comment by susannemoog** [comment]
> I think the two Points Developing & Using Extensions and then Developing with TYPO3 are kind of confusing. How can you Develop with TYPO3 without Extensions? Or is the second one about Core Development?

The second one contains TYPO3 Explained and the Changelog currently. Maybe a better name would be something like "TYPO3 Components" or similar - at least at the current state of TYPO3 Explained which isn't yet structured in line with the new plans.

**Comment by flicstar** [comment]
My feedback on the left-hand menu: I really like EVERYTHING except for one thing 😀 the 'Developing with TYPO3' menu.
Lina's already pointed it out and I'm not sure that renaming it with 'TYPO3 components' or something similar is the best option - but it certainly is an option. 
My suggestion would be to move both the Changelog and TYPO3 Explained under the Reference Manuals menu. I would have the Changelog listed first.
I would be inclined to rename TYPO3 Explained to 'TYPO3 A-Z' and list it last. What do you think?
ProposedMenu

File: Documentation/Home/ConfiguringTYPO3.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: RTE
+
+         .. container:: card-body
+
+            The heart of any content management system is its' editor. TYPO3 uses ckeditor
```
Comment (flicstar) [inline Documentation/Home/ConfiguringTYPO3.rst:134]:
```suggestion
            The heart of any content management system is its editor. TYPO3 uses ckeditor
```

File: Documentation/Home/ConfiguringTYPO3.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+         .. rst-class:: card-header
+
+            .. rubric:: Configuration Overview
+
+         .. container:: card-body
+
+            TYPO3 is well known for being extensive and easy to configure. The
+            configuration overview gives an overview of the various
+            configuration options available. It contains a description of the main
+            system configuration options.
```
Comment (flicstar) [inline Documentation/Home/ConfiguringTYPO3.rst:26]:
```suggestion
            The configuration overview gives an overview of the various
            configuration options available. It contains a description of the main
            system configuration options.
```

File: Documentation/Home/CreatingManagingContent.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
+.. include:: /Includes.rst.txt
+
+.. _creatingmanagingcontent:
+
+
+===========================
+Creating & Managing Content
+===========================
+
+The creation and management of pages and content are both handled within the Page module. Pages and page content can also be localized from the Page module.
```
Comment (flicstar) [inline Documentation/Home/CreatingManagingContent.rst:10]:
```suggestion
The creation and management of pages and content are both handled within the Page module. Pages and page content can also be localized from the Page module.
```
```suggestion

```
Can we remove this entirely?

File: Documentation/Home/CreatingManagingContent.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: Localizing Pages & Content
+
+         .. container:: card-body
+
+            The Localization Guide covers everything need to add additional languages to a TYPO3 site allowing content and pages
+            to be translated into an almost limitless number of languages.
```
Comment (flicstar) [inline Documentation/Home/CreatingManagingContent.rst:66]:
```suggestion
            The Localization Guide covers everything needed to add additional languages to a TYPO3 site, and how to translate content and pages.
```
Just removing some of the 'salesy' stuff.

File: Documentation/Home/CreatingManagingContent.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+         .. rst-class:: card-header
+
+            .. rubric:: Forms
+
+         .. container:: card-body
+
+            With the Form Framework, creating and customizing forms is easy.
+            The Form Framework allows non-technical as well technical users to create and extend complex web forms
+            directly from TYPO3's backend.
```
Comment (flicstar) [inline Documentation/Home/CreatingManagingContent.rst:88]:
```suggestion
            The Form Framework allows non-technical as well technical users to create and extend complex web forms
            directly from TYPO3's backend.
```
We want to keep thing objective, which means avoiding the word 'easy'.

File: Documentation/Home/ExtensionManuals.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: System Extensions
+
+         .. container:: card-body
+
+            Here you can find documentation of system extensions and third party extensions,
+            if available.
```
Comment (flicstar) [inline Documentation/Home/ExtensionManuals.rst:70]:
```suggestion
            Here you can find documentation for system extensions, as well as third party extensions,
            if available.
```

Comment (flicstar) [reply Documentation/Home/ExtensionManuals.rst:70]: Oh woops - you mean we're still missing docs for some system extensions? Ignore my suggestion then.

File: Documentation/Index.rst [hunk 7]
```diff
@@ -10,224 +16,252 @@ Welcome to the official TYPO3 Documentation
...
+         .. container:: card-body
 
-`Sitepackages
-<https://www.slideshare.net/benjaminkott/typo3-the-anatomy-of-sitepackages>`__
-allow you to bundle your Fluid templates and other site assets into a single,
-reusable extension that can be installed with a single click.
+            *  :ref:`Fluid is TYPO3’s templating engine. <t3coreapi:fluid>`
+               Fluid acts as the glue between your static HTML templates and the content you create in TYPO3’s backend.
 
+            * :ref:`Sitepackages <t3sitepackage:start>` allow you to bundle your Fluid templates and other site assets into a single, reusable extension that can be installed with a single click.
```
Comment (flicstar) [inline Documentation/Index.rst:88]:
```suggestion
            * :ref:`Sitepackages <t3sitepackage:start>` allow you to bundle your Fluid templates and other site assets into a single, reusable extension.
```
Feels a bit salesy - are we overpromising? You can keep it if you like.

File: Documentation/Index.rst [hunk 8]
```diff
@@ -10,224 +16,252 @@ Welcome to the official TYPO3 Documentation
...
+            .. rubric:: :ref:`Configuring TYPO3 <configuringTYPO3>`
 
-*  :ref:`Site Handling and Configuration <sitehandling-basics>` shows you how
-   to setup domains, languages, human-readable URLs and error pages.
+         .. container:: card-body
 
-*  :ref:`Backend User Management <t3start:user-management>` explains how you
-   setup backend users and grant them specific access to your installation of
-   TYPO3.
+            TYPO3 is well known for being extensive and easy to configure. The
```
Comment (flicstar) [inline Documentation/Index.rst:125]: Hmm, I commented on this elsewhere. Is it being pulled in here automagically, so I don't need to comment on it here as well?

**Review by flicstar — APPROVED** [review]
Great work +1

File: Documentation/Home/ExtensionManuals.rst (hunk 6 above)
Comment (susannemoog) [reply Documentation/Home/ExtensionManuals.rst:70]: Yes, sadly we do :/

**Comment by susannemoog** [comment]
> My feedback on the left-hand menu: I really like EVERYTHING except for one thing 😀 the 'Developing with TYPO3' menu.
> Lina's already pointed it out and I'm not sure that renaming it with 'TYPO3 components' or something similar is the best option - but it certainly is an option.
> My suggestion would be to move both the Changelog and TYPO3 Explained under the Reference Manuals menu. I would have the Changelog listed first.
> I would be inclined to rename TYPO3 Explained to 'TYPO3 A-Z' and list it last. What do you think?
> ProposedMenu

Thanks for the suggestions, I compromised ;) Both changelog and TYPO3 Explained are now in the references section (they were in the table there anyway, so that's fitting). I added TYPO3 Explained as the first item as imho it's the most important one. And I refrained from renaming it for now, as it's referenced with the name "TYPO3 Explained" everywhere (and to avoid too much confusion in this first step). Once we start restructuring / reworking that manual, we can see what name we want to choose.

As discussed, I'm going to merge this now so we can incrementally improve the pages in the next steps.

**Pull request merged**